tests/chainer_tests/functions_tests/activation_tests/test_rrelu.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+@testing.parameterize(*testing.product({
+    'train': [True, False],
+    'shape': [(3, 2), (5, 6)],
+    'dtype': [numpy.float16, numpy.float32, numpy.float64],
+}))
+class TestRReLU(unittest.TestCase):
+
+    def setUp(self):
+        # Avoid unstability of numeraical grad
+        self.x = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        self.x[(-0.05 < self.x) & (self.x < 0.05)] = 0.5
+        self.gy = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        # Asummption l < u
+        self.l = numpy.random.uniform(0, 1)
+        self.u = numpy.random.uniform(0, 1)
+        if self.l >= self.u:
+            self.l, self.u = self.u, self.l
+        self.check_forward_options = {}
+        self.check_backward_options = {}
+        if self.dtype == numpy.float16:
+            self.check_forward_options = {'atol': 1e-4, 'rtol': 1e-3}
+            self.check_backward_options = {'atol': 5e-4, 'rtol': 5e-3}
+
+    def check_forward(self, x_data):
+        x = chainer.Variable(x_data)
+        xp = cuda.get_array_module(x)
+        chainer.config.train = self.train
```

Comment:
Please use `with chainer.using_config('train', self.train):`, enclosing the `y = ...` line.

Reply:
I modified.